src/TwigComponent/src/ComponentPropertiesExtractor.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\TwigComponent;
+
+use Symfony\UX\TwigComponent\Attribute\ExposeInTemplate;
+use Symfony\UX\TwigComponent\Twig\PropsNode;
+use Twig\Environment;
+
+/**
+ * @author Jean-François Lépine <lepinejeanfrancois@gmail.com>
```

Comment:
As most of this class code comes from the original command code, this feels a bit over over-stating, wdyt ? :/

Reply:
I do not agree here since Jean-Francois is the creator of the file, he should put his name on the file

Reply:
![image](https://github.com/user-attachments/assets/f551a464-9d3e-4b3e-a9cc-7a37611a83be)

Reply:
Hi, 

Sorry! Indeed, I moved some code, and the header should mention both contributors. I didn't think of it. I'll address this along with the other feedback, if that's okay with you.

Reply:
Disagreements are rare around here, but always sincere. After discussing with my colleagues, I align with the general opinion. Thank you for the suggestion -not necessary after all- but appreciated!